util/ipgen/renderer.py

```diff
@@ -67,6 +67,11 @@ def get_template_parameter_values(
                         val_typed = int(val, 0)
                     else:
                         val_typed = val
+                elif template_param.param_type == 'bool':
```

Comment:
The # type in line 64 should add bool to the options in the Union. Better yet would be to make the type part of the declaration instead of a comment.

Reply:
I added the type in line 64. The more holistic approach though looks more intrusve.

Reply:
For future reference, I just checked you could add a variable declaration with type only above line 63.